apiary.apib

```diff
@@ -1708,6 +1708,17 @@ Get a list of all phases a deal can go through, sorted by their order in the flo
             + (object)
                 + id: `21efc56e-1ba8-469d-926a-e89502591b47` (string)
                 + name: `New` (string)
+                + deal_status: `new` (enum[string])
+                    + Members
+                        + new
+                        + open
+                        + won
+                        + lost
+                + triggers: (array)
```

Comment:
PP:
```suggestion
                + actions: (array[string])
```

Actions can simply be an array of enums right?

Reply:
Yes I think that would work fine.

Reply:
@jenssegers we were just going to wrap them in objects so we could add meta-data to actions later on, but I guess that's a simple BC-layer so, we can do that later on, indeed 🤷‍♂